exporter/exporterparser/stackdriver.go

```diff
@@ -84,3 +88,12 @@ func (sde *stackdriverExporter) ExportSpans(ctx context.Context, td data.TraceDa
 	// upload can use the context and information from the Node.
 	return exportSpans(ctx, "stackdriver", sde.exporter, td)
 }
+
+var _ exporter.MetricsExporter = (*stackdriverExporter)(nil)
+
+func (sde *stackdriverExporter) ExportMetricsData(ctx context.Context, md data.MetricsData) error {
+	for _, metric := range md.Metrics {
+		_ = sde.exporter.ExportMetric(ctx, md.Node, md.Resource, metric)
```

Comment:
I guess that the ExportMetric was defined to take only proto and stdlib types so no dependencies but it seems better to pass a slice of metrics instead of looping over them for every metrics exporter. Perhaps to be done separately.

Reply:
In deed, we can revisit that later after shipping this exporter out :)